tests/functional/spec/census-household.spec.js

```diff
@@ -96,18 +96,57 @@ describe('Census Household', function () {
         completeHouseholdAndAccommodation()
 
         // household-member
-        completeHouseholdDetails()
+        HouseholdMemberBegin.submit()
+        DetailsCorrect.clickDetailsCorrectAnswerYesThisIsMyFullName().submit()
+        Over16.clickOver16AnswerYes().submit()
+        PrivateResponse.clickPrivateResponseAnswerNoIDoNotWantToRequestAPersonalForm().submit()
+        Sex.clickSexAnswerMale().submit()
+        DateOfBirth.setDateOfBirthAnswerDay(2).setDateOfBirthAnswerMonth(3).setDateOfBirthAnswerYear(1980).submit()
+        MaritalStatus.clickMaritalStatusAnswerMarried().submit()
+        AnotherAddress.clickAnotherAddressAnswerNo().submit()
+        InEducation.clickInEducationAnswerNo().submit()
+        CountryOfBirth.clickCountryOfBirthEnglandAnswerEngland().submit()
+        Carer.clickCarerAnswerNo().submit()
+        NationalIdentity.clickNationalIdentityEnglandAnswerEnglish().submit()
+        EthnicGroup.clickEthnicGroupEnglandAnswerWhite().submit()
+        WhiteEthnicGroup.clickWhiteEthnicGroupEnglandAnswerEnglishWelshScottishNorthernIrishBritish().submit()
 
-        // visitors
-        completeVisitorSection()
+        expect(SexualIdentity.isOpen()).to.be.false
```

Comment:
For these it's worth adding a message otherwise it's not very clear in the logs when it fails like https://github.com/ONSdigital/eq-survey-runner/pull/660/files#diff-d1d932da779856122e7ba86f29c1e5c6R67